src/Workleap.DomainEventPropagation.Subscription.PullDelivery/Workleap.DomainEventPropagation.Subscription.PullDelivery.csproj

```diff
@@ -26,6 +26,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="OpenTelemetry.Api" Version="1.7.0" />
+    <PackageReference Include="System.Text.Json" Version="8.0.4" />
```

Comment:
It seems that the beta version of the Azure SDK were implicitly referencing a 4.7.4 version of System.Text.Json in those projects and even though it wasn't used previously in the project, I wonder if the correct decision is to explicitly reference the recent version here?

Reply:
synced offline and we'll move forward without the explicit addition. Given that this is a library, end users would need to patch it